website/frontend/templates/docs/scripting.html

```diff
@@ -152,12 +218,12 @@ <h3 id="in_x_y">$in(x,y)</h3>
           </li>
         </ul>
 
-        <h3 id="inmulti_x_y">$inmulti(x,y)</h3>
+        <h3 id="inmulti_x_y">$inmulti(%x%,y)</h3>
 
         <ul>
           <li>Returns true if multi-value variable
-            <code>x</code> contains
-            <code>y</code>.
+            <code>x</code> contains exactly
```

Comment:
This depends on https://github.com/metabrainz/picard/pull/710/files, right? So this change shouldn't go live until we have a stable version with that.